src/test/ui/regions/regions-bounded-by-trait-requiring-static.nll.stderr

```diff
@@ -1,45 +1,45 @@
 error: lifetime may not live long enough
-  --> $DIR/regions-bounded-by-trait-requiring-static.rs:22:5
+  --> $DIR/regions-bounded-by-trait-requiring-static.rs:26:5
    |
 LL | fn param_not_ok<'a>(x: &'a isize) {
    |                 -- lifetime `'a` defined here
 LL |     assert_send::<&'a isize>();
```

Comment:
It would be nice if we could point to `assert_send` and the `T: 'static` bound (non-NLL does this)

Reply:
I think this still needs and issue filed.

Reply:
Created https://github.com/rust-lang/rust/issues/96333